apps/sms/views/conversation/js/conversation.js

```diff
@@ -516,6 +547,11 @@ var ConversationView = {
 
     var prevPanel = args.meta.prev;
 
+    var emailThread = Settings.supportEmailRecipient &&
+      Threads.active.participants.some(Utils.isEmailAddress);
+
+    Compose.setupLock({ getType: () => emailThread ? 'mms' : null });
```

Comment:
Explained above why I call setup lock here even for non-email thread, alternatively it could be:
```
Compose.setupLock(emailThread ? { getType: () => 'mms' } : {});
```